### PR TITLE
Fixing accidental rename of server->bootstrap

### PR DESCRIPTION
**Pull request by nmittler**
These were introduced by #2739 (commit 49485f1c0787028e95e6de73cf597c333724b773)

### CODE REVIEW

File: tests/e2e/framework/rawVM.go [hunk 1]
```diff
@@ -211,7 +211,7 @@ func (vm *GCPRawVM) provision() error {
 			 --can-ip-forward \
 			 --service-account %s \
 			 --scopes "https://www.googleapis.com/auth/cloud-platform" \
-			 --tags "http-server","https-bootstrap" \
+			 --tags "http-server","https-server" \
```
Comment (kyessenov) [inline tests/e2e/framework/rawVM.go]: How did this pass the tests in the first place?

Comment (ldemailly) [reply tests/e2e/framework/rawVM.go]: that's sadly one of the "yet to be integrated" test (which also why mesh expansion broke in 0.3.0 etc...)

**Comment by kyessenov** [comment]
Well, if it's not tested, it's ain't ready :)

**Comment by kyessenov** [comment]
It's very unfortunate that we have no logs at all for the failed test.

**Comment by ldemailly** [comment]
there seems to be quite a lot of logging in
https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/2865/istio-pilot-e2e/1907/
but do you mean the artifacts link used to have pilot, ingress, etc logs ?

**Comment by kyessenov** [comment]
Correct, I'd like to see proxy logs on pod "a". There is a long standing
bug about loopback interfaces that I've been hunting for months, and proxy
logs are very helpful to have in these cases.

**Pull request merged**